### ISSUE TITLE
French translations

### ISSUE THREAD

**Issue by uportalis** [loomio/loomio#3898]
In the global e-mail settings:
"Inscrivez-vous pour participer" ("Subscribe on participation") actually means "Subscribe in order to participate". It could be changed to "En cas de participation" ("In case of participation") or "Si je participe" ("If I participate").

The proposal closure comment (e.g. "Close in 11 minutes") is partially translated (e.g. "Fermeture à in 11 minutes", which means "Closure at in 11 minutes"). Not sure it is easy to change as the string "in xx minutes" seems to not be localized).

The menu item "Démarrer un nouveau groupe" ("Start a new group") is not fully displayed in the left menu (see the attached screenshot).

The option item "Gérer les inscriptions au groupe" ("Manage the subscriptions to the group", or "Manage group subscription" in EN locale) actually points to a page to select the pricing scheme. Maybe it could be changed to "Gérer l'option tarifaire" ("Choose your pricing plan").

The "Preview" tab is not translated (should be "Aperçu"). And "No content to preview" should be "Aucun contenu à afficher" ("No content to display").

**Comment by uportalis** [comment]
![loomio_start_a_group](https://cloud.githubusercontent.com/assets/11533685/20596284/11e9633c-b23e-11e6-9948-394e188ce985.PNG)

**Comment by gdpelican** [comment]
Thanks @uportalis . Are you part of the Loomio translation community on [Transifex](https://www.transifex.com/rdbartlett/loomio-1)? That would be the place to raise these questions, and propose changes to the existing translations.

Some of the strings you mentioned are part of plugins, which were currently working on allowing translations for; it'll be done before the end of the year.

**Comment by swergas** [comment]
@uportalis rejoins-nous sur transifex pour améliorer les traductions :)

* In the global email settings page, I'm not sure what the best translation would be for "Subscribe on participation". Maybe "S'abonner aux notifications lorsque je participe ?"

* I just looked at the "Fermeture à in 11 minutes" problem, and the original translation string is Closing {{time}}", where {{time}} seems to be replaced by a duration which is always written in English, so it's mixing languages. Maybe the loomio developers can make the application show the duration in the language of the rest of the UI? Anyway, a better translation could be "Fermera {{time}}", I just changed it in transifex.

* I also see the "Démarrer un nouveau groupe" string cut in the menu, as well as for some group names which are listed in my menu.

* Yes, for "Gérer l'option tarifaire du groupe", it seems to be a translation mistake (original is "Manage Group Subscription"). I just replaced it by "Gérer l'option tarifaire du groupe", thanks :).

* For "Preview" and "No content to display", these are very probably the strings which are part of a not-yet-translatable plugin @gdpelican is talking about, as they don't seem to appear in the list of translatable strings on transifex.
